src/System.Management.Automation/utils/PInvokeDllNames.cs

```diff
@@ -13,124 +13,6 @@ namespace System.Management.Automation
     /// </summary>
     internal static class PinvokeDllNames
     {
-#if CORECLR
```

Comment:
>RECLR [](start = 6, length = 5)

We need this code as well for PS to work on Nano Server. For Inventory agent/take over LCM, we need to rely on #else part of the code.